src/capi/azext_capi/_help.py

```diff
@@ -106,6 +106,9 @@
   - name: --vnet-name
     type: string
     short-summary: Name of the Virtual Network to create
+  - name: --wait-for-nodes
+    type: bool
```

Comment:
What do you think about making this an `int` instead that defaults to 0? That would allow the user to override `control_plane_machine_count + node_machine_count` which may be incorrect for templates with multiple node pools or for user-supplied templates.